src/pudl/metadata/resources/ferc1.py

```diff
@@ -131,6 +131,32 @@
         "etl_group": "ferc1",
         "field_namespace": "ferc1",
     },
+    "income_statement_ferc1": {
+        "description": "Income statement.",
+        "schema": {
+            "fields": [
+                "record_id",
+                "utility_id_ferc1",
+                "report_year",
+                "utility_type",
+                "income_type",
+                "income",
+                "balance",
+                "ferc_account",
```

Comment:
the `ferc_account` is merged in from the xbrl metadata, but does not always have a value because in this table the `income_type` does not always correspond to a FERC account.

In the PIS table we have a `ferc_account_label` is the categorical(-ish) column/`xbrl_factoid` that was stacked. So the labels in that table come directly from the XBRL column names.